app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    caching_place = {}
+
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
The return type for `wrapper` should be not `None`, but rather `Any`, because you can apply it to any function.